### ISSUE TITLE
[YouTube] Videos take a while to load and buffer frequently on some resolutions

### ISSUE THREAD

**Issue by b1k3rdude** [TeamNewPipe/NewPipe#10183]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.1

### Steps to reproduce the bug

Open any subscribed video and its will take a while to load the thumbnail and then after loading it will get stuck on buffering after a few seconds.

### Expected behavior

Open any subscribed video and it almost immediatly load the thumbnail and then start playing the video.

### Actual behavior

Open any subscribed video and its will take a while to load the thumbnail and then after loading it will get stuck on buffering after a few seconds.

### Screenshots/Screen recordings

no screenshot/recording its a streaming/buffering issue.

### Logs

no log, its a streaming/buffering issue.

### Affected Android/Custom ROM version

Android 11

### Affected device model

S20FE 

### Additional information

This only stated happening as of UK lunchtime yesterday.

**Comment by Taratect** [comment]
This was a previous issue which has now resurfaced. I have previously posted this bug on #6424 . Now for me it only buffers while viewing in 720p and buffering goes away when changing any other resolution even higher ones. One connection I found is that the videos that has 60 fps format are having this problem in 720p. However 720p60 is working without buffering. Dk if it's related. And changing to 3gp or WebM is a temporary band aid which may not work always

**Comment by IMMalebolgia** [comment]
I can confirm that on the Fire Stick 4k Max the buffering issue happens at 720p/30, as that is my default resolution for YouTube stuff, and when I switch to 1080p the problem goes away. This is a reasonable workaround but I tend to run up against my 1.28TB monthly limit so hopefully the 720p issue can be resolved.

**Comment by b1k3rdude** [comment]
I typically watch videos at 1080p, but and see this issue at 720p and 1080p 30 or 60. My current wor around is to download the videos, but to get decent d/l speeds i ha ebto use all 33 threads on my s20fe.

**Comment by b1k3rdude** [comment]
Ok so today I thought I would see if the issue is still present, and to my suprise videos are no longer buffering. So what ever YT broke a few days ago it appears to have been fixed?

I will continue to test till the end of the week, and then mark the issue as resolved if applicable etc.

**Comment by WMT-IT** [comment]
I'm also experiencing the buffering issue with 720p on both of my devices.
Going to 1080p/60 does indeed fix the issue for me as well.

**Comment by laurenjapes** [comment]
Experiencing the same issue, though I'm glad I checked here for a workaround. Changing the video quality is also fixing the issue on my end.

**Comment by Zlopez** [comment]
I have the similar issue, just see the comments being loaded and then nothing, even the network activity stops. The change to other resolution for streams doesn't help.

**Comment by b1k3rdude** [comment]
Ok so "I think" I found whats causing the bufferuing for me atleast, and I did so by accident.

I have "adaway" installed for blocked ads, with this enabled newpipe will as per the OP gets stuck on buffering. If I close newpipe and disable adaway and open newpipe the buffering issue goes away. 

Downloading videos works regardless if adaway is enabled/disabled, however I think download speeds are reduced with adaway enabled.
